src/types.ts

```diff
@@ -511,6 +515,27 @@ export interface InitialAccountCredential {
     contents: InitialCredentialDeploymentValues;
 }
 
+export type BakerPendingChange =
+    | {
+          change: 'ReduceStake';
+          newStake: bigint;
```

Comment:
You have to make sure this is correctly parsed as a `bigint` when parsing the JSON.